linux_os/guide/system/auditing/auditd_configure_rules/audit_file_deletion_events/audit_rules_file_deletion_events.rule

```diff
@@ -54,3 +54,14 @@ ocil: |-
     {{{ ocil_audit_syscall(syscall="renameat") }}}
 
 {{{ ocil_clause_entry_audit_syscall() }}}
+
+warnings:
+    - general: |-
+        This rule aims to attest collection of events from multiple syscalls
+        related to file deletion. It is written with DISA STIG in mind, other
+        policies should use separated rules for each syscall, namely:
```

Comment:
@yuumasato I don't think that we need to mention the rules below as it becomes another maintenance burden if this list changes. Let's either remove the mentioned audit rules, or change `, namely:` to `. For example:`

Reply:
@shawndwells We should also consider scenario of someone tailoring a Profile and adding these "combined" audit rules, they won't see the developer warning.